conf/config.properties

```diff
@@ -21,7 +21,7 @@ users.password.regex.tooltip=Enter a combination of atleast six characters
 users.admin.localonly=false
 
 # SMTP emailing switch (for password recovery and signup verification etc)
-smtp.mails.enabled=true
```

Comment:
Don't remove an existing field

Reply:
Actually, this is an implication, I think, we don't require it